src/MagnumPlugins/WebPImporter/Test/CMakeLists.txt

```diff
@@ -0,0 +1,74 @@
+#
+#   This file is part of Magnum.
+#
+#   Copyright © 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019,
+#               2020, 2021, 2022 Vladimír Vondruš <mosra@centrum.cz>
+#
+#   Permission is hereby granted, free of charge, to any person obtaining a
+#   copy of this software and associated documentation files (the "Software"),
+#   to deal in the Software without restriction, including without limitation
+#   the rights to use, copy, modify, merge, publish, distribute, sublicense,
+#   and/or sell copies of the Software, and to permit persons to whom the
+#   Software is furnished to do so, subject to the following conditions:
+#
+#   The above copyright notice and this permission notice shall be included
+#   in all copies or substantial portions of the Software.
+#
+#   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+#   THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+#   FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+#   DEALINGS IN THE SOFTWARE.
+#
+
+# IDE folder in VS, Xcode etc. CMake 3.12+, older versions have only the FOLDER
+# property that would have to be set on each target separately.
+set(CMAKE_FOLDER "MagnumPlugins/WebPImporter/Test")
+
+# CMake before 3.8 has broken $<TARGET_FILE*> expressions for iOS (see
+# https://gitlab.kitware.com/cmake/cmake/merge_requests/404) and since Corrade
+# doesn't support dynamic plugins on iOS, this sorta works around that. Should
+# be revisited when updating Travis to newer Xcode (xcode7.3 has CMake 3.6).
+if(NOT MAGNUM_WEBPIMPORTER_BUILD_STATIC)
+    set(WEBPIMPORTER_PLUGIN_FILENAME $<TARGET_FILE:WebPImporter>)
+endif()
+
+# First replace ${} variables, then $<> generator expressions
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/configure.h.cmake
+               ${CMAKE_CURRENT_BINARY_DIR}/configure.h.in)
+file(GENERATE OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG>/configure.h
+    INPUT ${CMAKE_CURRENT_BINARY_DIR}/configure.h.in)
+
+corrade_add_test(WebPImporterTest WebPImporterTest.cpp
+    LIBRARIES Magnum::Trade
+    FILES
+        empty.webp
+        nature.png
+        tampered_penguin.webp
+        animated.webp
+        RGB_lossless.webp
```

Comment:
From the naming convention in other tests, these should be `rgb-lossless.webp`, `tampered-penguin.webp` for example, also called [`kebap-case`](https://betterprogramming.pub/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841) :)

These conventions make it easier to decide on how to name files and remember/guess how they were named.

Here's a unix command for linux/macos/msys (git bash on windows) to rename all the webp files:
```sh
for f in *.webp; do mv $f "$(echo $f | tr [A-Z] [a-z] | tr _ -)"; done
```

Reply:
I've changed the variable and file names, as well as commenting style after thoroughly reading Corrade Coding Style a couple of times but all those rules are a bit overwhelming and will take time to get used to :) So, I might have made some mistakes again.

Reply:
No worries, these things do indeed take time :)